mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx

```diff
@@ -20,13 +21,23 @@ export const TagStats = (props: TagStatsProps) => {
   return props.stats.map((tagStat) => (
     <Fragment key={tagStat.name}>
       <StatisticLine
-        description={`${tagStat.name} jobs finished:`}
+        description={<JobTag
+          key={tagStat.name}
+          jobTag={tagStat.name}
+          isSmall
+        />}
         value={`${tagStat.totalAmount} (${tagStat.totalAmountPercentage}%)`}
+
       />
       <StatisticLine
         key={tagStat.name}
-        description={`${tagStat.name} time:`}
+        description={<JobTag
+          key={tagStat.name}
```

Comment:
And here?)

Reply:
And here was the key before me

Reply:
ok check it too

Reply:
In this context, this is done as a warning from react: Each child in a list should have a unique "key" prop